_includes/filezilla.md

```diff
@@ -1,9 +1,31 @@
 ## General instructions
 
-The raw data of all studies published in IDR can be downloaded using using anonymous File Transfer Protocol (FTP). We recommend using the [FileZilla client](https://filezilla-project.org/download.php).
+The raw data of all studies published in IDR can be downloaded using [Globus](https://www.globus.org/) or anonymous File Transfer Protocol (FTP).
 
 You can download both whole studies as well as parts/single images.
 
+We recommend using the [Globus](https://www.globus.org/).
+
+## Globus
+
+Download and install [Globus connect personal](https://www.globus.org/globus-connect-personal).
+
+You might have to create an account, or you can use other login providers like ORCiD.
+
+Go to "Web: Transfer files" to get to the Web UI.
+
+<img src="img/filezilla/globus_1.png" alt="Globus Web UI" />
+
+Then in the File Manager / Collections search for "EMBL-EBI Public Data".
+
+<img src="img/filezilla/globus_2.png" alt="Globus Collection search" width="50%" />
+
+In there you should be able to navigate through the same directory structure like on the FTP server /pub/databases/IDR/ to get to the IDR submission you want to download. 
+
+<img src="img/filezilla/globus_3.png" alt="Globus IDR collection" width="50%" />
+
+## FTP
```

Comment:
I think it is fine to have an FTP header.
But I think you need to also work on the "Desktop client step-b-step" header below: the Desktop step-by-step is a subheader of the FTP header (afaik) and this must be clear (at the moment it looks like a third option -> confusing.